root/help.md

```diff
@@ -0,0 +1,41 @@
+% DHCP-CLIENT (1) Container Image Pages
+% Karsten Hopp
```

Comment:
I tried to get some clearance from Adam Miller, and it seems that they prefer it would be container maintainer [1]: "For Fedora, it's supposed to be the Container Maintainer. We don't currently have the concept of a "Help File Maintainer" "

[1] [https://mojo.redhat.com/docs/DOC-1101797#comment-1068294](https://mojo.redhat.com/docs/DOC-1101797#comment-1068294)